Code/GraphMol/ChemReactions/testReaction.cpp

```diff
@@ -775,8 +775,8 @@ void test8Validation() {
   rxn = RxnSmartsToChemicalReaction(smi);
   TEST_ASSERT(rxn);
   /* 08/08/14
-   * This test is changed due to allowing same atom mapping muliple times in the
-   * products */
+   * This test is changed due to allowing same atom mapping multiple times in
```

Comment:
This comment is not meaningful to me. Not sure about others.

Reply:
I'll remove it.